### PR TITLE
`Tutorial groups`: Add documentation

### PR DESCRIPTION
**Pull request by stefanwaldhauser**
#### General
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/guidelines/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context
- Motivation: Missing documentation about tutorial group feature
### Description
- Added a documentation about the tutorial group feature
- Linked to this documentation in `about us` and `readme`
- The documentation includes a video tutorial for instructors, tutors and students respectively.

### Steps for Testing
- Just read through documentation and watch the videos - Link: https://artemis-platform--6387.org.readthedocs.build/en/6387/

#### Code Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

File: docs/user/tutorialgroups.rst [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Overview
+--------
+Artemis facilitates the coordination of tutorial groups in a course. Tutorial groups are a learning strategy where students teach and learn from each other in small groups (20-30). In this strategy, proficient students act as tutors and lead the groups. The tutor and the group members usually meet weekly either on campus or online. Students present their solutions to homework assignments or other tasks and receive feedback and suggestions from the tutor and their peers.
+
+.. note:: The tutorial group overview page (used by students) displays dates in the students's current time zone. The tutorial group management page (used by instructors and tutors to manage groups and their sessions) displays dates in the course time zone. This is helpful if the instructors or tutors travel and work across multiple time zones. A header at the top of the page shows the current time zone used for dates.
```
Comment (b-fein) [inline docs/user/tutorialgroups.rst:14]:
```suggestion
.. note:: The tutorial group overview page (used by students) displays dates in the student's current time zone. The tutorial group management page (used by instructors and tutors to manage groups and their sessions) displays dates in the course time zone. This is helpful if the instructors or tutors travel and work across multiple time zones. A header at the top of the page shows the current time zone used for dates.
```

File: docs/user/tutorialgroups.rst [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+Viewing Tutorial Groups as a Student
+------------------------------------
+
+.. raw:: html
+
+    <iframe src="https://live.rbg.tum.de/w/artemisintro/27043?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
+        Watch this video on TUM-Live.
+    </iframe>
+
+Students can access their tutorial grouos for a specific course by navigating to the course's ``Tutorial Group`` page.
```
Comment (b-fein) [inline docs/user/tutorialgroups.rst:122]:
```suggestion
Students can access their tutorial groups for a specific course by navigating to the course's ``Tutorial Group`` page.
```

File: docs/user/tutorialgroups.rst [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+|tutor-registrations|
+
+Managing the Sessions of a Tutorial Group
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By clicking on the ``Sessions`` button, the tutor can view the list of sessions that are scheduled for the tutorial group. From this list, the tutor can cancel or reschedule sessions. The tutor can also create new sessions by clicking on the ``Create New Tutorial Group Session`` button. Furthermore, the tutor can enter the number of attending students for each session. This information is used to calculate the utilization of the tutorial group.
+
+.. note::
+    The utilization of a tutorial group is the average attendance divided by capacity (if defined). The average attendance considers the last three sessions. If no attendance is entered, the corresponding session is ignored and the calculation is performed with two or one session.
```
Comment (danielbode) [inline docs/user/tutorialgroups.rst:108]:
```suggestion
    The utilization of a tutorial group is the average attendance divided by capacity (if defined). The average attendance considers the last three sessions. If no attendance is entered, the corresponding session is ignored and the calculation is performed with one or two sessions.
```

File: docs/user/tutorialgroups.rst [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+* **Artemis managed tutorial group channels:** This option allows Artemis to create and manage a dedicated channel for each tutorial group in the 'Messages' section of the course. This feature is only selectable if the course has the ``Messaging`` feature enabled in the course settings. If activated, tutorial group channels can still be managed manually but Artemis automatically performs some common tasks, such as:
+
+  * Adding and removing students from the channel when they register or unregister for the tutorial group
+
+  * Making the assigned tutor a moderator of the channel
+
+  * Deleting the channel when the tutorial group is deleted
+
+If not all of these configurations are done, the instructor will see a checklist page with the missing configurations. These configurations can also be changed later in the ``Global Configuration`` section of the tutorial group page. A link to this section is hidden behind the ``More...`` button.
```
Comment (b-fein) [inline docs/user/tutorialgroups.rst:46]:
```suggestion
If not all of these configurations are done, the instructor will see a checklist page with the missing configurations. These configurations can also be changed later in the ``Global Configuration`` section of the tutorial group page. A link to this section can be accessed via the ``More...`` button.
```
To me ‘hidden feature’ usually sounds like an euphemism for a bug or bad UI.

**Review by b-fein — APPROVED** [review]

**Pull request merged**